wolox-books/src/store.js

```diff
@@ -20,4 +21,32 @@ export default new Vuex.Store({
       context.commit('setUserStatus', false)
     }
   }
+}
+
+const moduleBooks = {
```

Comment:
Since we are creating different modules, each module should be a separated file from each other in a folder called `modules`